crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -438,17 +451,33 @@ fn convert_block_response_to_sequencer_block_data(
 }
 
 #[instrument(skip_all)]
-async fn submit_blocks_to_data_availability_layer(
-    client: CelestiaClient,
+async fn submit_blocks_to_celestia(
+    client: celestia_client::jsonrpsee::http_client::HttpClient,
+    fee: u64,
+    gas_limit: u64,
     sequencer_block_data: Vec<SequencerBlockData>,
     validator: Validator,
 ) -> eyre::Result<u64> {
+    use celestia_client::{
+        celestia_types::blob::SubmitOptions,
+        CelestiaClientExt as _,
+    };
+
     info!(
         num_blocks = sequencer_block_data.len(),
         "submitting collected sequencer blocks to data availability layer",
     );
-    let rsp = client
-        .submit_all_blocks(sequencer_block_data, &validator.signing_key)
-        .await?;
-    Ok(rsp.height)
+    let height = client
+        .submit_sequencer_blocks(
+            SEQUENCER_NAMESPACE,
+            sequencer_block_data,
+            &validator.signing_key,
+            SubmitOptions {
+                fee: Some(fee),
+                gas_limit: Some(gas_limit),
```

Comment:
what happens if we don't put a fee or gas limit? does the node/client select them for us?

Reply:
I actually don't know. I have submitted this: https://github.com/astriaorg/astria/issues/510